src/js/components/dropdownButton.js

```diff
@@ -1,34 +1,19 @@
-document.addEventListener('DOMContentLoaded', () => {
-  const mobileLink = document.querySelector('.dropdown-mobile-toggle > button');
-  const mobileDropdown = document.querySelector('.mobile-dropdown');
-  const desktopLink = document.querySelector('.dropdown-desktop-toggle > button');
-  const desktopDropdown = document.querySelector('.desktop-dropdown');
+const behavior = require('uswds/src/js/utils/behavior');
 
-  function addListenerMulti(el, s, fn) {
-    s.split(' ').forEach((e) => el.addEventListener(e, fn, false));
-  }
+function createDropdownPicker(contentSelector) {
+  return function onDropdownButtonClickOrKeyPress(event) {
+    const button = event.target;
+    const content = button.parentNode.querySelector(contentSelector);
+    const isExpanded = !content.classList.toggle('display-none');
```

Comment:
Does USWDS polyfill `classList` already for us?

Reply:
> Does USWDS polyfill `classList` already for us?

Yeah:

https://github.com/uswds/uswds/blob/92ae6f8/src/js/polyfills/index.js#L1-L2